lib/sprockets/commoner/processor.rb

```diff
@@ -7,14 +7,15 @@ class Processor < Schmooze::Base
 
       ExcludedFileError = Class.new(::StandardError)
 
+      VERSION = '1'.freeze
       BABELRC_FILE = '.babelrc'.freeze
       PACKAGE_JSON = 'package.json'.freeze
       JS_PACKAGE_PATH = File.expand_path('../../../js', __dir__)
       ALLOWED_EXTENSIONS = /\.js(?:on)?(?:\.erb)?\z/
 
       dependencies babel: 'babel-core', commoner: 'babel-plugin-sprockets-commoner-internal'
 
-      method :version, 'function() { return [process.version, babel.version]; }'
```

Comment:
Removed this because the node version shouldn't influence the output of the processor